vertical-pod-autoscaler/pkg/admission-controller/Makefile

```diff
@@ -83,9 +83,16 @@ build-in-docker-%: clean docker-builder
 	echo '=============== bulding from the above ==============='
 	docker run -v `pwd`/../..:/gopath/src/k8s.io/autoscaler/vertical-pod-autoscaler vpa-autoscaling-builder:latest bash -c 'cd /gopath/src/k8s.io/autoscaler/vertical-pod-autoscaler && make build-binary-with-vendor-$* -C pkg/admission-controller'
 
+.PHONY: create-buildx-builder
+create-buildx-builder:
+	BUILDER=$(shell docker buildx create --use)
```

Comment:
Is this guaranteed to create a docker-container or should this be specified explicitly?

Reply:
Good catch, I've added a commit to set the driver explicitly to `docker-container`!

Reply:
Done!